bot/cogs/jams.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from discord import Member, PermissionOverwrite, utils
+from discord.ext import commands
+
+from bot.constants import Roles
+from bot.decorators import with_role
+
+log = logging.getLogger(__name__)
+
+
+class CodeJams:
+    """
+    A cog for managing the code-jam related parts of our server
```

Comment:
cog is implied. prefer `Code jam related management tasks`

Reply:
```suggestion
    Manages the code-jam related parts of our server
```